### PR TITLE
Keb suspension

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- enables Suspension/unsuspensions triggered by ERS context update

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/broker/instance_update.go [hunk 1]
```diff
@@ -57,6 +65,25 @@ func (b *UpdateEndpoint) Update(ctx context.Context, instanceID string, details
 		logger.Info(k)
 	}
 
+	if b.processingEnabled {
+		err = b.contextUpdateHandler.Handle(instance, ersContext)
```
[the remark this answers is not in the document]
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/internal/broker/instance_update.go:69]: good point, let's discuss

File: docs/kyma-environment-broker/03-12-suspension.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+           "subaccount_id": "subaccount-id",
+           "active":false
+       },
+       "parameters": {
+           "name": "cluster-name"
+       }
+   }'
+```
+
+Kyma Environment Broker exposes a `runtimes/` endpoint which allows us to check SKRs.
```
[the remark this answers is not in the document]
Comment (piotrmiskiewicz) [reply docs/kyma-environment-broker/03-12-suspension.md]: I'll create the doc once again from the scratch

File: components/kyma-environment-broker/internal/broker/instance_get.go [hunk 3]
```diff
@@ -27,26 +26,18 @@ func NewGetInstance(instancesStorage storage.Instances, log logrus.FieldLogger)
 // GetInstance fetches information about a service instance
 //   GET /v2/service_instances/{instance_id}
 func (b *GetInstanceEndpoint) GetInstance(ctx context.Context, instanceID string) (domain.GetInstanceDetailsSpec, error) {
-	logger := b.log.WithField("instanceID", instanceID)
```
[the remark this answers is not in the document]
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/internal/broker/instance_get.go:30]: reverted

File: components/kyma-environment-broker/internal/process/input/builder.go [hunk 4]
```diff
@@ -106,6 +107,7 @@ func (f *InputBuilderFactory) CreateProvisionInput(pp internal.ProvisioningParam
 		return nil, errors.Errorf("case with plan %s is not supported", pp.PlanID)
 	}
 
+	logrus.Infof("init provisioning input")
```
[the remark this answers is not in the document]
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/internal/process/input/builder.go]: not necessary, removed

**Review by ksputo — DISMISSED** [review]

**Comment by ksputo** [comment]
please add `UpdateProcessingEnabled` to `values.yaml` and KEB deployment - but you can do that in separate PR too

**Review by ksputo — APPROVED** [review]

**Pull request merged**